### ISSUE TITLE
Empty object when importing css

### ISSUE THREAD

**Issue by luxferoo** [symfony/webpack-encore#402]
Hi, I want to import a css file "css-modules way" but when I do this I got an empty object.

`import test from './test.scss' // test == {}`

**Comment by Lyrkan** [comment]
Hi @luxferoo,

Could you try this?

```js
Encore.configureCssLoader(config => {
    config.modules = true;
});
```

**Comment by luxferoo** [comment]
@Lyrkan Hello my friend, Encore.configureCssLoader is not a recognized property or method

**Comment by Lyrkan** [comment]
Oh, my bad I just noticed that it hasn't been released yet (added in #335), for some reason I thought it was way older than that.

So... currently there is no "easy" way to do it.
You can still modify the generated Webpack config object though, doing something like this should work:

```js
const Encore = require('@symfony/webpack-encore');

Encore
    .setOutputPath('public/build/')
    .setPublicPath('/build')
    // (...)
;

// Retrieve the config object
const config = Encore.getWebpackConfig();

// Set "modules" option of the css-loader
for (const rule of config.module.rules) {
    if (rule.use) {
        for (const ruleUse of rule.use) {
            if (ruleUse.loader === 'css-loader') {
                ruleUse.options.modules = true;
            }
        }
    }
}

// Export the config (don't call getWebpackConfig() again)
module.exports = config;
```

**Comment by luxferoo** [comment]
@Lyrkan thanks buddy !
